packages/tx/test/inputValue.spec.ts

```diff
@@ -136,3 +146,30 @@ tape('[Transaction Input Values]', function (t) {
     st.end()
   })
 })
+
+tape('[Invalid Array Input values]', (t) => {
+  const txTypes = [0x0, 0x1, 0x2]
+  for (const txType of txTypes) {
+    const tx = TransactionFactory.fromTxData({ type: txType })
```

Comment:
OK this is a great test. Shall we generalize this and do a check for all keys which we do not want array?

Bonus: we should also check for the access lists, that in the access list the account is not array. Also for storage array: the elements should be keys and not arrays. (I think we do not check this ATM)

Reply:
I thought it already did, unless I'm missing something. It creates a transaction with default values and then overrides one by one with an array. Unless I'm imagining things, that should hit all of the `txData` interface properties which shouldn't be arrays.